dist/boards/slstk3400a/doc.txt

```diff
@@ -22,19 +22,19 @@ actively measure the power consumption of your hardware and code, in real-time.
 | RAM             | 8.0KiB                                               |
 | Flash           | 64.0KiB                                              |
 | EEPROM          | no                                                   |
-| Frequency       | up to 24 MHz                                         |
+| Frequency       | up to 25 MHz                                         |
 | FPU             | no                                                   |
 | MPU             | no                                                   |
-| DMA             | 8 channels                                           |
-| Timers          | 3x 16-bit + 1x 16-bit (low power)                    |
+| DMA             | 6 channels                                           |
+| Timers          | 2x 16-bit + 1x 16-bit (low power)                    |
```

Comment:
It does have 3x 16-bit timers.

Reply:
Okay. Referring to section 3.5 Product Overview in [EFM32HG-RM](https://www.silabs.com/documents/public/reference-manuals/efm32hg-rm.pdf) shows that there is no LE Timer. So should it be just 3 x 16 Bit timer ?

Reply:
You are right, I'll update this tonight.

Reply:
Updated.